packages/@react-aria/link/src/useLink.ts

```diff
@@ -16,6 +16,7 @@ import {filterDOMProps, mergeProps, shouldClientNavigate, useRouter} from '@reac
 import React, {RefObject} from 'react';
 import {useFocusable} from '@react-aria/focus';
 import {usePress} from '@react-aria/interactions';
+import {withBasename} from '../../utils/src/openLink';
```

Comment:
Please delete this line, add `useBasename` to the import for `@react-aria/utils` on line 15. Then add `useBasename` to the export for `./openlink` on line 20 of `@reat-aria\utils\index.ts`.

Reply:
Wasn't sure if you wanted another function exported from the utils package or not, so thanks for the guidance. Updated the PR.